calculator.c

```diff
@@ -0,0 +1,74 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <string.h>
+
+int isOperator(char c){  
+    return c == '+' || c == '-' || c == '*' || c == '/';
+}
+
+int calculator(int a, int b, char op){
```

Comment:
function names should describe what the function does. rename it to something more descriptive.